### PR TITLE
Linear response calculations with range-separated functional

### PR DESCRIPTION
**Pull request by tomaskubar**
includes calculation of forces, and changes to QM/MM interface
Replaces pull request #381

### CODE REVIEW

File: prog/dftb+/lib_timedep/rs_linresp.F90 [hunk 1]
```diff
@@ -0,0 +1,3188 @@
...
+  end subroutine getSOffsite
+
+
+  !> Set value of sizeIn to new, twice as large, value
+  pure subroutine incSizeInt(sizeIn)
+
+    integer, intent(inout) :: sizeIn
+    sizeIn = 3 * sizeIn
+
+  end subroutine incSizeInt
```
[the remark this answers is not in the document]
Comment (thomas-niehaus) [reply prog/dftb+/lib_timedep/rs_linresp.F90:148]: True, @tomaskubar can you comment?

[the remark this answers is not in the document]
Comment (tomaskubar) [reply prog/dftb+/lib_timedep/rs_linresp.F90:148]: I don't know why that is. Also I'm not sure what would be better - factor 2 or 3 :-/ Maybe you can decide?

[the remark this answers is not in the document]
Comment (thomas-niehaus) [reply prog/dftb+/lib_timedep/rs_linresp.F90]: @tomaskubar: This was coded by Julian, not by you? 
I updated the comments. Code works for now. We will have to look deeper into the algo to find best choices here.

[the remark this answers is not in the document]
Comment (tomaskubar) [reply prog/dftb+/lib_timedep/rs_linresp.F90]: I thought, either you or Julian. Myself, I did very little coding - only what was needed for the refactoring to new DFTB+.
Sure, this is one of the things that need cleanup. I think there may be a few more.

**Pull request merged**